[Repository] Documentation updated (READMES and folders). Closes #2 (#3)
Hernancano98/Project-Book: Java/README.md

> [<-- BACK TO ROOT](../README.md)

Reviewer: ```suggestion
[<-- Back to root](../README.md)
```

# **Java**

Java is a compiled programming language, class-based and object-oriented, which is most used for backend and mobile development.

> The **main objective** with this language is to create software that has a desktop version and a mobile app version, and both versions communicate with the same database. The following diagram shows the idea that motivates the learning of this language:

Reviewer: I'm not sure if that's the main objective of Java, but in case that you're right, mobile and desktop apps communicate with a backend server (Java main purpose), which communicates with a database, front-end does **not** communicate directly to a database.

```suggestion
The **main objective** with this language is to create software that has a desktop and a mobile app, and both communicate with the same backend. The following diagram shows the idea that motivates the learning of this language:
```

![Structure](./docs/img/Structure.png)

> Right now we are in the learning and practice stage, but the goal is on track.

Reviewer: What is the goal???

Author: The goal is to build something like these:

Create working software that has a desktop version and a mobile app version, and both versions communicate with the same database.

Reviewer: Then I'll make another suggestion

```suggestion
Right now we are in the learning and practice stage, but the goal is to create working software that has a desktop and a mobile version, and both communicate with the same back-end service.
```

Author: If we put that sentence, we will be rewriting the paragraph above the image, please look at the screenshots i left in the PR description. 

This is the paragraph i'm talking about.

![image](https://user-images.githubusercontent.com/68404335/88868629-12b93680-d1e7-11ea-8310-f100fbb05139.png)

Reviewer: You're right, keep it the way it is.

> ### Projects:

Reviewer: Why do you go from a # to a ###? your skipping h2
```suggestion
## Projects:
```

> + [REST API](./HernanCano/API_REST/README.md) - Hernán Cano

Reviewer: Decide if you are going to call it REST API or API REST, you shouldn't call the link REST API and the directory API_REST

```suggestion
+ [REST API](./HernanCano/API_REST/README.md)
```

> ### Sub-directories:

Reviewer: Why do you go from a # to a ###? your skipping h2
```suggestion
## Sub-directories:
```

+ [Enzo Avena](./EnzoAvena/README.md)
+ [Hernán Cano](./HernanCano/README.md)

> [<-- BACK TO ROOT](../README.md)

Reviewer: ```suggestion
[<-- Back to root](../README.md)
```
